### ISSUE TITLE
get oneTooneIndex data

### ISSUE THREAD

**Issue by uxiaveleiro** [MedicineBiology-AI/EEG-DTI#4]
How do you generate the files in 'oneTooneIndex' folder?
https://github.com/MedicineBiology-AI/EEG-DTI/blob/24073650a72e6cba12122ca3bef5a310782bd967/decagon/deep/minibatch.py#L87

**Comment by GeniusYx** [comment]
Thanks for your interest.
 We divide the known DTIs into train set and test set in advance.
 The specific code can use the built-in method of scikit-learn.
 The example code is as follows:
`
from sklearn.model_selection import KFold
#all_edge_idx represents the all known DTIs.
all_edge_idx = list(range(edges_all.shape[0]))
np.random.seed(123)
np.random.shuffle(all_edge_idx)
# Then we use the 10-flod cross-validation method to envaluate our mode, we first generate the train set and test set.
kf = KFold(n_splits=10, shuffle=True, random_state=123)
for pos_train_idx, pos_test_idx in kf.split(all_edge_idx):
    # pos_train_idx is train set, pos_test_idx is test set
`
I hope my reply can help you.

**Comment by GeniusYx** [comment]
Thanks for your interest.
We divide the known DTIs into train set and test set in advance.
The specific code can use the built-in method of scikit-learn.
The example code is as follows:
```
from sklearn.model_selection import KFold
#all_edge_idx represents the all known DTIs.
all_edge_idx = list(range(edges_all.shape[0]))
np.random.seed(123)
np.random.shuffle(all_edge_idx)
#Then we use the 10-flod cross-validation method to evaluate our mode, we first generate the train set and test set.
kf = KFold(n_splits=10, shuffle=True, random_state=123)
for pos_train_idx, pos_test_idx in kf.split(all_edge_idx):
      # pos_train_idx is train set, pos_test_idx is test set
```
I hope my reply can help you.

**Comment by uxiaveleiro** [comment]
ok, thanks
